api/v1alpha1/etcdcluster_types.go

```diff
@@ -41,6 +41,9 @@ type EtcdClusterSpec struct {
 	// +optional
 	PodDisruptionBudgetTemplate *EmbeddedPodDisruptionBudget `json:"podDisruptionBudgetTemplate,omitempty"`
 	Storage                     StorageSpec                  `json:"storage"`
+	// Security describes security settings of etcd (authentication, certificates, rbac)
+	// +optional
+	Security *SecuritySpec `json:"security,omitempty"`
```

Comment:
This seems incosistent. Why should `Security` be a pointer to `SecuritySpec`, but `Storage` a `StorageSpec` literal?

Reply:
Looks like storage also should be pointer if we allow to create resource without specified storage

Reply:
Storage was implemented some time ago - the first implementation. Also we have a bug in storage spec. So storage is not an example